DesignSafe-CI/portal
# task/DES-2759: Apps side nav styling

## Overview: ##

## PR Status: ##


## Related Jira tickets: ##

* [DES-2759](https://tacc-main.atlassian.net/browse/DES-2759)

## Summary of Changes: ##
- Applied `navLink` styles for links in side nav via correcting CSS class name
- Applied active and hover background color for nav elements via antd's Menu theme tokens
- Delineated side nav from rest of app form by adding side barrier (to match design)
- Added `shortLabel` as first option for rendering an app's label in the side nav

## Testing Steps: ##
1. Go to https://designsafe.dev/rw/workspace/padcirc?appVersion=55.01 (or any app page, as convenient)
2. Confirm the hover + selected color for nav items matches the screenshot below
3. Confirm there is now a dark border on the right side of the side nav as shown in the screenshot below
4. _shortLabel test pending_

## UI Photos:
<img width="1636" alt="image" src="https://github.com/DesignSafe-CI/portal/assets/43251554/e5a68221-dbeb-400d-b065-824c75007c46">


## Notes: ##
Please review code additions relating to rendering front end - unsure what I am missing to capture the shortLabel for the adcirc-frontera app.
